internal/experimental/registry/client.go

```diff
@@ -220,6 +220,7 @@ func (c *Client) Pull(ref string, options ...PullOption) (*PullResult, error) {
 	store := content.NewMemoryStore()
 	allowedMediaTypes := []string{
 		ConfigMediaType,
+		LegacyChartContentLayerMediaType,
```

Comment:
This should be appended under `if operation.withChart {` below
```
allowedMediaTypes = append(allowedMediaTypes, ChartLayerMediaType, LegacyChartLayerMediaType)
```

Reply:
ack